rust/platforms/BUILD

```diff
@@ -0,0 +1,25 @@
+# See https://forge.rust-lang.org/platform-support.html
```

Comment:
This BUILD file doesn't seemed used anywhere, remove?

Reply:
Removed for now. The platform decls are intended to parallel @bazel_tools//platforms:* and be used as a convenience for declaring intent to cross compile, but they're not useful now.